lib/storage/data/file/utils.js

```diff
@@ -0,0 +1,34 @@
+const fs = require('fs');
+const posixFadvise = require('fcntl');
+
+/**
+ * Release free cached pages associated with a file
+ *
+ * @param {String} filePath - absolute path of the associated file
+ * @param {Int} fd - file descriptor of the associated file. If null,
+ *    filePath will be opened to get the file descriptor
+ * @param {werelogs.RequestLogger} log - logging object
+ * @return {undefined}
+ */
+function releasePageCacheSync(filePath, fd, log) {
+    function release(filePath, fd, log) {
+        const ret = posixFadvise(fd, 0, 0, 4);
+        if (ret !== 0) {
+            log.warning(
+            `error fadv_dontneed ${filePath} returned ${ret}`);
+        }
+    }
+    if (fd === null) {
```

Comment:
does this happen ?

Reply:
No, but added it as a feature for later use.

Reply:
yes please remove if not needed

Reply:
done!